src/components/gds-label/_index.scss

```diff
@@ -3,6 +3,13 @@
   text-transform: uppercase;
 }
 
+@mixin size-xl {
+  font-size: var(--label-xl-font-size);
+  line-height: 19px;
+  letter-spacing: var(--label-xl-letter-spacing);
+  font-weight: var(--label-xl-font-weight);
+}
```

Comment:
Not ideal to keep introducing sizes. Currently I can't use `size-l` because the menu uses it. Would probably be better if `--menu-item-font-weight` uses a generic label (no size), and then set the `--label-font-size` or something.

Reply:
I don't mind having an xl label but yeah, maybe it might be better for the menu item to override those variables?